packages/account/src/providers/provider.ts

```diff
@@ -2106,7 +2106,7 @@ Supported fuel-core version: ${supportedVersion}.`
   /**
    * @hidden
    */
-  private validatePaginationArgs(params: {
+  validatePaginationArgs(params: {
```

Comment:
You can make this static, but it's okay like this as well.

Reply:
I plan to externalize this as a helper. I did not do that right because it involved moving some tests around. Do you think it is better to make it static instead?

Reply:
A helper function is even better. The problem is that removing this method later down the line would constitute a breaking change now that you made it public. I think you can move it to a helper function right away. You can write the tests for it later if you want. The most important thing is that you added the test for `getTransactionsSummaries` in this PR.

Reply:
@nedsalk https://github.com/FuelLabs/fuels-ts/pull/3400/commits/0ad5fd65f0d1e9627db648160505f01d1d09dec7